site/content/blog/2021-06-01-whats-new-in-svelte-june-2021.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What's new in Svelte: June 2021
+description: Progress on SvelteKit V1 and tighter TypeScript/Svelte integrations in language tools
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+This month, we saw lots of contributions to SvelteKit and its docs. The language tools also got some new features alongside some hydration bug fixes in Svelte core. Let's jump into the updates...
```

Comment:
"some hydration bug fixes in Svelte core" - I might just leave this part out here. It was a perf improvement that was added and then reverted because it was broken. I think the mention down below is fine though